src/queries/__tests__/role.test.tsx

```diff
@@ -77,3 +78,61 @@ test('getAllByRole, queryAllByRole, findAllByRole', async () => {
     getNoInstancesFoundMessage(NO_MATCHES_TEXT)
   );
 });
+
+describe('*ByRole with a name', () => {
+  test('Find an element that has the corresponding role and a children with the name', () => {
+    const { getByRole } = render(
+      <TouchableOpacity accessibilityRole="button" testID="target-button">
+        <Text>Save</Text>
```

Comment:
duplicate the test, but duplicate the <Text> inside the <TO>, like this:

```ts
<TouchableOpacity accessibilityRole="button" testID="target-button">
  <Text>Save</Text>
  <Text>Save</Text>
</TouchableOpacity>
```

This way we can check that it does not throw on finding two matching `name` instances inside component with riven role